README.md

```diff
@@ -1,11 +1,82 @@
+![Aragon](https://res.cloudinary.com/dacofvu8m/image/upload/v1677353961/Aragon%20CodeArena/osx_blue_logo_lqrvkr.png)
+
 # Aragon JS SDK
 
-This folder contains all the JS packages available on NPM:
+Welcome to the Aragon JS SDK! This SDK allows you to interact with Aragon OSx
+using JavaScript. Whether you are a developer wanting to build on top of Aragon,
+or looking to contribute to the SDK itself, you are in the right place! This
+folder contains all the JS packages available on NPM:
 
 - [Client](./modules/client)
 - [IPFS](./modules/ipfs)
 - [Common](./modules/common)
 
+## Developer Quick Start
+
+The Aragon OSx SDK is Javascript agnostic, which means you can use it with any
+Javascript framework, including popular ones like React, Vite, or Vue. The
+examples in this readme are written using NodeJS
+
+First things first install the client with
+
+```sh
+yarn add @aragon/sdk-client
+```
+
+or
+
+```sh
+npm install @aragon/sdk-client
+```
+
+Then, you'll want to set up the Aragon OSx SDK context within your application
+to have access to the SDK functions.
+
+```typescript
+// 1. Create a context
+const minimalContextParams = {
+  network: "mainnet",
+  web3Providers: "https://rpc.ankr.com/eth",
+  signer: Wallet.createRandom(),
+};
+
+const context = new Context(minimalContextParams);
+```
+
+Next thing you'll want to do is set up the general purpose client so you can
+call on the SDK functions. This client is used to interact with any DAO on the
+network you're connected to.
+
+```typescript
+const client = new Client(context);
+```
+
+Now test its all working by fetching an array of DAOs
+
+```typescript
+console.log(await client.methods.getDaos({ limit: 5 }));
+```
+
+returns
+
+```typescript
+[
```

Comment:
Let's be more explicit with the return example we get from the SDK. A better example would be something like:

```js
{
    address: "0xf2d594f3c93c19d7b1a6f15b5489ffce4b01f7da",
    ensDomain: "your-dao.dao.eth",
    metadata: {
      name: "Your DAO",
      description: "Description about your DAO",
      avatar: "https://ipfs.avatar.your-dao.com",
    },
    plugins: [{
        sub-domain: "plugin.your-dao.eth",
        .... // add the right example attributes here  
    }]
};
```